swagger.json

```diff
@@ -2607,8 +2781,86 @@
                     "format": "double"
                 }
             }
-        },          
-                                    
+        },
+        "NAWebhookCameraEvent": {
+            "properties": {
+                "app_type": {
+                    "type": "string",
+                    "enum": [
+                        "app_camera"
```

Comment:
Since this comes from Netatmo, I think we shouldn't use an enum here for evolution compatibility. Eg. if Netatmo sends a new app_type you might still want to deserialize the event and maybe log it even if the application code doesn't do anything with it. Zalando's guidelines suggest to use an extension x-extensible-enum instead.